src/libstd/sys/windows/backtrace.rs

```diff
@@ -157,3 +160,37 @@ unsafe fn _write(w: &mut Write) -> io::Result<()> {
 
     Ok(())
 }
+
+fn query_full_process_image_name() -> io::Result<PathBuf> {
+    super::fill_utf16_buf(|buf, mut sz| unsafe {
+        c::QueryFullProcessImageNameW(c::GetCurrentProcess(), 0, buf, &mut sz); sz
```

Comment:
Probably a bug they fixed in Windows 8/10 or something. Windows... 😛 

I converted it to a `PathBuf`  for the call to `OpenOptions::open` below, in a way it's a function that queries for a path so it makes sense that it returns a `Path[Buf]`.